ambry-commons/src/test/java/com.github.ambry.commons/TestUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+
+class TestUtils {
+
+  /**
+   * Awaits on the passed-in {@link CountDownLatch}. If times out throws an exception.
+   * @param latch The latch to await on.
+   * @param timeoutMs Timeout in millisecond.
+   * @throws Exception If awaits for more than the specified time, throw a {@link TimeoutException}.
+   */
+  static void awaitLatchOrTimeout(CountDownLatch latch, long timeoutMs) throws Exception {
```

Comment:
This method seems generic enough that it can go to `com.github.ambry.utils.TestUtils`. You don't need to create a new class here for just this reason (and can create it when there is a real need).